src/fieri/app/models/metrics_runner.rb

```diff
@@ -24,4 +24,8 @@ def cookbook_api_response(params)
   def cookbook_version_api_response(params)
     SupermarketApiRunner.new.cookbook_version_api_response(params['cookbook_name'], params['cookbook_version'])
   end
+
+  def source_repo(cookbook_json)
+    cookbook_json.match(%r{(?<=github.com\/)[\w-]+\/[\w-]+}).to_s
```

Comment:
I get the feeling that this makes `MetricsRunner` have to know too much about implementation details of the various metric workers. Maybe we could consider a `SourceRepoMetric` class from which both `ContributingFileWorker` and the soon-to-exist `TestingFileWorker` could descend. That parent class could contain the logic for looking up the source repository URL from `cookbook_json` and could provide a shared, private `octokit_client` to hide the details of how to determine "source contains a thingie."

Speaking of looking up the source repo URL, I'd prefer a JSON.parse-gimmie-['source_url'] over a capturing regex treating the JSON response like a string.